source/_integrations/husqvarna_automower_ble.markdown

```
@@ -0,0 +1,46 @@
+ha_integration_type: integration
+ha_domain: husqvarna_automower_ble
+---
+
+The Husqvarna Automower BLE integration provides connectivity with Husqvarna Automowers lawn mowers via a local Bluetooth connection. This allows connecting and controlling an Automower without any accounts, cloud or network connection.
+
+The integration is based on [AutoMower-BLE](https://github.com/alistair23/AutoMower-BLE), a unofficial reverse engineered Husqvarna Automower Connect BLE library.
+
+### Tested an verified mowers
+
+This has been tested against the following mowers, although it is expected that all will work
+ - Automower 305
```

Markdownlint

Expected: 0; Actual: 1
Unordered list indentation

(MD007, ul-indent)